doc/news/DM-45761.feature.rst

```diff
@@ -0,0 +1,4 @@
+- - Introduced a suite of scripts for taking sensitivity matrices and parameter marches with LSSTCam and LSSTComCam:
```

Comment:
Instead of adding multiple entries in one file, create one individual file for each entry. For each additional file after the first one, append a number to the file name, e.g.;

- doc/news/DM-45761.feature.rst
- doc/news/DM-45761.feature.rst.1
- doc/news/DM-45761.feature.rst.2

Reply:
Although, it looks like the Towncrier tests are complaining now...